src/dbnode/storage/bootstrap/bootstrapper/peers/options.go

```diff
@@ -55,7 +55,7 @@ var (
 	// src/cmd/services/m3dbnode/config package if this is changed.
 	DefaultShardPersistenceFlushConcurrency = 1
 	// defaultIndexSegmentConcurrency defines the default index segment building concurrency.
-	defaultIndexSegmentConcurrency = 1
+	defaultIndexSegmentConcurrency = int(math.Min(2, float64(runtime.NumCPU())))
```

Comment:
There's also a similar thing in `fs/options.go`, should they be kept the same? `indexSegmentConcurrency` config argument seems to be setting both of them.

https://github.com/m3db/m3/blob/922a33becf9079d6228f6e06130da39e2edbad5d/src/dbnode/storage/bootstrap/bootstrapper/fs/options.go#L62-L63

Also, does this ask for a release notes entry?

Reply:
@vpranckaitis yes ah, can we export this as `DefaultIndexSegmentConcurrency` then reference this value from the `fs/options.go`?

We definitely need a single place for the default to live.